zebrad/src/commands/start.rs

```diff
@@ -385,6 +400,8 @@ impl StartCmd {
                     exit_when_task_finishes = false;
                     Ok(())
                 }
+
+                // TODO: add scan task which is tricky because it needs to be behind a feature.
```

Comment:
I was not able to make a version for the scanner inside the `select!` macro that works behind the `zebra-scan` feature. Any input is appreciated.

Reply:
What compiler error do you get when you try it, and what code have you tried?

I'll make a suggestion above which might fix it, but it's hard to be sure without the actual code and error messages.